assets/js/modules/search-console/dashboard/dashboard-widget-sitestats.js

```diff
@@ -45,12 +45,18 @@ class SearchConsoleDashboardWidgetSiteStats extends Component {
 
 	setOptions() {
 		const { selectedStats, series, vAxes } = this.props;
+		const { pageTitle } = global.googlesitekit;
 
-		const pageTitle = global.googlesitekit.pageTitle && global.googlesitekit.pageTitle.length ? sprintf( __( 'Search Traffic Summary for %s', 'google-site-kit' ), decodeHtmlEntity( global.googlesitekit.pageTitle ) ) : __( 'Search Traffic Summary', 'google-site-kit' );
+		let title = __( 'Search Traffic Summary', 'google-site-kit' );
+
+		if ( pageTitle && pageTitle.length ) {
+			/* translators: %s: page title */
+			title = sprintf( __( 'Search Traffic Summary for %s', 'google-site-kit' ), decodeHtmlEntity( pageTitle ) );
+		}
```

Comment:
The ESLint rules weren't properly detecting the translators string in a multiline ternary so I split it up this way, which is probably cleaner anyway 😄

Reply:
Makes sense 👍 

With the ternary it wouldn't be clear which `__()` call it should be associated with.